src/kili/services/plugins/upload.py

```diff
@@ -279,6 +293,9 @@ def _check_plugin_runner_status(self, update=False):
             if status == "ACTIVE":
                 break
 
+            if status == "FAILED":
```

Comment:
It would be a bit cleaner to just add ` or status == "FAILED"` two lines above

Reply:
indeed 😅